clients/cli.go

```diff
@@ -38,3 +42,32 @@ func (c *CLI) PrintTable(headers []string, rows ...map[string]interface{}) error
 	}
 	return nil
 }
+
+func (c *CLI) GetOrgId(ctx context.Context, paramOrgId, paramOrgName string, orgApi api.OrganizationsApi) (string, error) {
+	if paramOrgId != "" {
+		return paramOrgId, nil
+	}
+	orgName := paramOrgName
+	if orgName == "" {
+		orgName = c.ActiveConfig.Org
+	}
+	if orgName == "" {
+		return "", ErrMustSpecifyOrg
+	}
+	res, err := orgApi.GetOrgs(ctx).Org(orgName).Execute()
```

Comment:
This has a small behavior change now from a few of the commands, like `secret` which. would query `GetOrgs(ctx)` for first the passed parameter, then the active config if none found, but in this case it will only use the active config if no user parameter is specified. I'm not sure which is the "better" option but something to point out I think for discussion

Reply:
I much prefer not having to know the state of the InfluxDB instance in order to know how the command will behave, and I don't think the old behaviour is documented, so I strongly feel the updated version is better.

Reply:
Thanks for pointing out the behaviour change though!